docs/sources/mimir/reference-configuration-parameters/index.md

```diff
@@ -1700,21 +1702,12 @@ sharding_ring:
   # CLI flag: -alertmanager.sharding-ring.heartbeat-timeout
   [heartbeat_timeout: <duration> | default = 1m]
 
-  # (advanced) The replication factor to use when sharding the alertmanager.
-  # CLI flag: -alertmanager.sharding-ring.replication-factor
-  [replication_factor: <int> | default = 3]
-
-  # (advanced) True to enable zone-awareness and replicate alerts across
-  # different availability zones.
-  # CLI flag: -alertmanager.sharding-ring.zone-awareness-enabled
-  [zone_awareness_enabled: <boolean> | default = false]
-
   # (advanced) Instance ID to register in the ring.
   # CLI flag: -alertmanager.sharding-ring.instance-id
   [instance_id: <string> | default = "<hostname>"]
 
-  # (advanced) List of network interface names to look up when finding the
-  # instance IP address.
+  # List of network interface names to look up when finding the instance IP
```

Comment:
This was `(advanced)`. I think we lost it in the refactoring.

Reply:
Yes, it was, but inconsistently so (i.e. it was not marked `(advanced)` for the other components). I didn't see a reason why it should be advanced only for alertmanager and not for any of the other components using this?

Reply:
Ok. I didn't notice while doing my review. Then you did the right thing 👍